packages/modules/pricing/src/services/pricing-module.ts

```diff
@@ -1295,3 +1221,24 @@ export default class PricingModuleService
     })
   }
 }
+
+const hashPrice = (
+  price: PricingTypes.PriceDTO | PricingTypes.CreatePricesDTO
+): string => {
+  const data = Object.entries({
+    currency_code: price.currency_code,
+    price_set_id: "price_set_id" in price ? price.price_set_id : null,
+    price_list_id:
+      "price_list_id" in price ? price.price_list_id ?? null : null,
+    min_quantity: price.min_quantity ?? null,
+    max_quantity: price.max_quantity ?? null,
+    ...("price_rules" in price
+      ? price.price_rules?.reduce((agg, pr) => {
+          agg[pr.attribute] = pr.value
+          return agg
+        }, {})
+      : {}),
+  }).sort(([a], [b]) => a.localeCompare(b))
+
+  return crypto.createHash("md5").update(JSON.stringify(data)).digest("hex")
```

Comment:
https://github.com/medusajs/medusa/blob/develop/packages/core/utils/src/common/simple-hash.ts#L2

Reply:
Nice, changed to it now